server/app.js

```diff
@@ -1,7 +1,11 @@
-const express = require('express');
-const app = express();
-const server = require('./index.js')(app);
+'use strict';
 
-app.listen(3000, () => {
+var express = require('express');
+var app = express();
+// eslint-disable-next-line no-unused-vars
+var server = require('./index.js')(app);
```

Comment:
Took this

https://github.com/simplabs/ember-simple-auth/blob/84ca953ec7ac2f3db41ebb378b3c4d68208a64bd/server/.eslintrc.js#L3

as a cue that node 0.x support is important here for some reason. I'd also be happy to update the `.eslintrc.js` in this folder to permit node features that weren't available in `0.x` if that's the desirable way to address this conflict

Reply:
We're still supporting Ember 1.12 🙀 - I'm not sure actually that implies we **have** to support Node 0 as well…